src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,21 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+
+@Dao
```

Comment:
we don't need @Dao here

Reply:
@IvanZuravlev228 is right, it is simply redundant here, we need this annotation for identifying the implementations for the interfaces